### ISSUE TITLE
contact

### ISSUE THREAD

**Issue by codeforequity-at** [flix-tech/robotmate#18]
hi, i am looking for interesting use cases for botium (i am the project lead), and this project certainly is one. is this in production use ? (if its allowed to ask ...)

**Comment by lleites** [comment]
Yes, it is used on CI/CD of Flixbus bots and voice integration services.
